examples/aditof-demo/aditofdemoview.cpp

```diff
@@ -797,11 +801,11 @@ void AdiTofDemoView::render() {
         }
 
         if (captureEnabled || captureBlendedEnabled) {
-            if (temp_cnt++ > (displayFps < 10) ? 10 : 30) {
+            if (temp_cnt++ > ((displayFps < 10) ? 10 : 30)) {
                 std::pair<float, float> temp = m_ctrl->getTemperature();
                 temp_cnt = 0;
-                sprintf(afe_temp_str, "AFE TEMP: %.1f", temp.first);
-                sprintf(laser_temp_str, "LASER TEMP: %.1f", temp.second);
+                sprintf_s(afe_temp_str, "AFE TEMP: %.1f", temp.first);
```

Comment:
is sprintf_s available on linux? As far as I know it's windows only.

Reply:
True. I made a mistake when I edited the commits: I had changes for Linux and Windows in the same file (this one and others), and I added this file to the Linux commit.